src/hotspot/os/posix/os_posix.cpp

```diff
@@ -537,6 +537,28 @@ bool os::has_allocatable_memory_limit(julong* limit) {
     *limit = (julong)rlim.rlim_cur;
     result = true;
   }
+
+#if defined(INCLUDE_TSAN) && defined(AARCH64)
```

Comment:
INCLUDE_TSAN is always defined, it has a value of 0 or 1 (see make/hotspot/lib/JvmFeatures.gmk).
Probably "#if INCLUDE_TSAN && defined(AARCH64)".